### PR TITLE
Style changes. Legend first badge not working.

### PR DESCRIPTION
**Pull request by MaanavD**
Please review before merging. Below is an image of the updated table.
image

### CODE REVIEW

**Comment by justinchuby** [comment]
Do we want to maintain the patterns on the cells to make the site more accessible?

**Comment by justinchuby** [comment]
Looks like the supported color is different than the actual color used. Otherwise very nice change. Thanks!

File: op-vis/package.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.1.3",
+    "bootstrap": "^5.2.2",
```
Comment (justinchuby) [inline op-vis/package.json:12]: We may be able to remove the bootstrap css line in 

https://github.com/justinchuby/torch-onnx-op-matrix/blob/cc705d7032f4fe7df4874ba703536a8d31773fc2/op-vis/public/index.html#L26-L31

then?

File: op-vis/src/App.scss [hunk 2]
```diff
@@ -0,0 +1,89 @@
+
```
Comment (justinchuby) [inline op-vis/src/App.scss:1]: Is `App.css` no longer useful and can be removed with this?

Comment (MaanavD) [reply op-vis/src/App.scss:1]: Yeah, .scss just takes over all the styling

File: op-vis/package.json (hunk 1 above)
Comment (MaanavD) [reply op-vis/package.json:12]: Will remove the css line!

File: op-vis/src/App.scss (hunk 2 above)
Comment (justinchuby) [reply op-vis/src/App.scss:1]: Could you help delete App.css? Thanks!

**Review by justinchuby — APPROVED** [review]
LGTM thanks!

**Pull request merged**